src/ui/templates/results/resultssectionheader.hbs

```diff
@@ -35,7 +35,7 @@
           data-eventtype="FILTERING_WITHIN_SECTION"
           data-eventoptions='{{eventOptions}}'
         >
-          {{#if _config.changeFiltersText}}{{_config.changeFiltersText}}{{else}}change{{/if}}
+          {{#if _config.changeFiltersText}}{{_config.changeFiltersText}}{{else}}{{translate phrase='change' }}{{/if}}
```

Comment:
Same question about context for this one, could this be confused for something like money change?